ddocs/medic-client/views/contacts_by_freetext/map.js

```diff
@@ -32,9 +32,10 @@ function(doc) {
 
   var types = [ 'district_hospital', 'health_center', 'clinic', 'person' ];
   var idx = types.indexOf(doc.type);
-  var dead = !!doc.date_of_death;
   if (idx !== -1) {
-    var order = dead + ' ' + idx + ' ' + (doc.name && doc.name.toLowerCase());
+    var dead = !!doc.date_of_death;
```

Comment:
Can we `const` in map functions?

Reply:
Apparently not https://github.com/scdf/eslint-plugin-couchdb#extra-rules

Reply:
Trying to manually edit view map functions in couch directly, I got various results: it allowed me to use `const`, but not `let` and no arrow functions. So I guess we'll stick yo ES5.